### DISCUSSION TITLE
关于毕业以及奖励标准

### DISCUSSION THREAD

**Discussion by jolestar** [starcoinorg/web3startrek#89] [General]
这个问题我们在 https://github.com/movefuns/web3startrek/pull/61  https://github.com/movefuns/web3startrek/pull/59 讨论过，初步达成了共识。现在 MoveFuns 目标是构造跨 Move 多链生态的学习社区，同时为了鼓励大家参与，想把毕业标准分级，需要重新制定一下标准。

所以有以下提议：

### 基础要求

* 技术需要涉及 Move + (Web|Rust|Other Language).

### L1 毕业

* 通过学习日志以及练习项目证明自己掌握了 Move 语言
* 奖励 200$ USDT 或者等值 STC

### L2 毕业

* 给生态贡献了新的独立的 Move modules.
* 对外的 Move 相关的教程文章, 希望 Move 被更多的人了解，所以鼓励通过其他渠道传播文档教程.
* 对 Move 生态项目（[move-language](https://github.com/move-language),[movefuns](https://github.com/movefuns)）有 PR 贡献，mentor 会评估 PR 的复杂度.
* 以上三个项目需要满足二项
* 额外奖励 300$ USDT 或者等值 STC

### 生态项目额外奖

* 对 [starcoinorg](https://github.com/starcoinorg) 下的项目有重要的 PR 贡献：额外奖励 500$ USDT 或者等值 STC
* 对 [OLSF](https://github.com/OLSF) 下的项目有重要的 PR 贡献：(TODO)
* 对 [aptos-labs](https://github.com/aptos-labs) 下的项目有重要的 PR 贡献：(TODO)
* 对 [MystenLabs](https://github.com/MystenLabs) 下的项目有重要的 PR 贡献：(TODO)

**Comment by nangongamo** [comment]
挺好，毕业要求更具象、更细化了，可以自行评估难易程度选择合适自己的毕业类别。

**Comment by leeduckgo** [comment]
赞！

**Comment by v1xingyue** [comment]
感觉毕业标准 还是不够具体，应该再展开一些:  比如 ： 

- 了解 Move 语法
-  能编写部署合约
- 能编写完整的dapp 
-  ....

另外，每个点都可以展开链接到对应的学习资料。
这样就可以形成一个完整的Dao学习系统了。
